### PR TITLE
Create script to run the application for devs and production 

### PR DESCRIPTION
**Pull request by ThomasBracher**
* Initialize database at launch of index.js
* Using `retry` npm to retry the connection to database
* `run.sh` launches the application for dev and production environments
* `dev-run.sh` for dev purposes (tailf on the application logs after relaunch)

remark: init_storage.js might be useless now that its done at application startup

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+    operation.attempt(function () {
+        storage.execute(script, function(err) {
+            if (operation.retry(err)) {
+                console.log(err.message);
+                return ;
+            }
+            console.log('Database initialized');
+        });
+    });
+});
```
Comment (thomaslule) [inline index.js]: why this block? database shouldn't be emptied each time the server is restarted

Comment (ThomasBracher) [inline index.js]: The script is modified to only create the table if it does not exists.

Comment (ThomasBracher) [inline index.js]: Oups forgot to remove the *drop* line :confounded:

Comment (thomaslule) [inline index.js]: yes :) and anyway the aim of this script __is__ to reinitialize the database by dropping everything inside

Comment (ThomasBracher) [inline index.js]: added the *initialize* script and the *drop all* one and refactored `init_storage.js` accordingly.

**Pull request merged**